linebot/imagemap.go

```diff
@@ -41,6 +41,17 @@ type ImagemapArea struct {
 	Height int `json:"height"`
 }
 
+// ImagemapVideo type
+type ImagemapVideo struct {
+	OriginalContentURL string       `json:"originalContentUrl"`
+	PreviewImageURL    string       `json:"previewImageUrl"`
+	Area               ImagemapArea `json:"area"`
+	ExternalLink       *struct {
+		LinkUri string `json:"linkUri"`
```

Comment:
We are using `golint` for unifying coding styles.

```
linebot/imagemap.go:50:3: struct field LinkUri should be LinkURI
```

Reply:
done

Reply:
Thank you very much!